### ISSUE TITLE
urls in pages have http/https in them making it hard to move from http to https or place the site behind https offload

### ISSUE THREAD

**Issue by mrgum** [eprints/eprints3.4#405]
I'd like eprints pages **not** to have http / https in their links I'd prefer either like

/cgi/request_doc

or if needed (though I do not see why it would be)

//eprints.domain.com/cgi/request_doc

having the http/s in the pages makes it harder to host eprints (and makes the pages longer than they need be :))  There is quite a bit of logic in eprints that could just be removed as I don't think its useful anymore.

when I say harder to host I mean harder to move from http to https or place the site behind https offload

**Comment by drn05r** [comment]
EPrints is often embedded in other website so using paths rather the full URLs is not always possible.  We did investigate in the past moving to protocol relative URLs (e.g. //eprints.example.org/path) but this just complicated matters further as it was yet another type of URL that could be used.

We have tried to dig deep into how URLs are generated and see if they could all be replaced with protocol relative URLs but this was not something we felt could be safely achieved in a minor release.  As the number of files where the code would need to be modified was excessive.  We will added this requirement to the list for the next major release of EPrints.  As we can refactor the code to reduce the number of locations where code changes will be required.
